dev/ansible/03_run_tests.yml

```diff
@@ -5,16 +5,57 @@
     return_content: yes
     status_code: 200
     body_format: json
+  when: lookup('env', 'consul_acl_default_policy') != 'deny'
   register: result_minio
   retries: 15
   delay: 15
   until: result_minio.json | length == 2
   tags: test
 
+- name: Minio healthchecks pass with token
+  uri:
+    url: http://localhost:8500/v1/health/checks/minio?filter={{ '(Status=="passing")' | urlencode }}
+    method: GET
+    headers:
+      X-Consul-Token: "master"
+    return_content: yes
+    status_code: 200
+    body_format: json
+  when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
+  register: result_minio
+  retries: 15
+  delay: 15
+  until: result_minio.json | length == 2
+  tags: test
+
+- name: Check if intention allow exists for minio-local and minio
+  shell: consul intention match -source minio-local
+  register: intention_exist
+  when: lookup('env', 'consul_acl') | bool and lookup('env', 'consul_acl_default_policy') == 'deny'
+  tags: test
+
+- name: Consul create intention allow for minio-local and minio
+  shell: nohup consul intention create minio-local minio
+  register: test_output_intention
+  when: intention_exist is defined and intention_exist != {}
+  environment:
+    CONSUL_HTTP_TOKEN: "master"
+  tags: test
```

Comment:
Can we move it to `dev/ansible/00_create_intention_allow_mc_minio.yml`?

Reply:
This set of code is specific and required only when we run tests so i thought of keeping it near/in the same file where we have the tests.
The 00_create_intention_allow_mc_minio.yml file holds code for creating intention between mc and minio, which we require for the mc nomad job and should be run before terraform apply is executed, so it is placed separately without any tags.